packages/pds/src/app-view/api/app/bsky/feed/getFeedSuggestions.ts

```diff
@@ -0,0 +1,29 @@
+import { Server } from '../../../../../lexicon'
+import AppContext from '../../../../../context'
+
+export default function (server: Server, ctx: AppContext) {
+  server.app.bsky.feed.getFeedSuggestions({
+    auth: ctx.accessVerifier,
+    handler: async ({ req, auth, params }) => {
+      const requester = auth.credentials.did
+
+      if (await ctx.canProxyRead(req, requester)) {
```

Comment:
sweet yup 👍 

while we still have the appview proxy logic in here, we'll need to toggle the knob that enables proxying this

just remind us to do it when deploying

Reply:
Oh, I suppose I can just remove this check entirely, huh